src/main/scala/tech/cryptonomic/conseil/application/LorreBakerRightsOperations.scala

```diff
@@ -0,0 +1,85 @@
+package tech.cryptonomic.conseil.application
+
+import tech.cryptonomic.conseil.util.IOUtils.{lift, IOLogging}
+import tech.cryptonomic.conseil.config.LorreAppConfig
+import tech.cryptonomic.conseil.tezos.{TezosDatabaseOperations => TezosDb}
+import tech.cryptonomic.conseil.tezos.TezosTypes.{discardGenesis, BlockMetadata}
+import cats.effect.{IO, Timer}
+import scala.{Stream => _}
+import com.typesafe.scalalogging.LazyLogging
+import cats.implicits._
+import com.typesafe.scalalogging.Logger
+import scala.concurrent.ExecutionContext
+import akka.stream.ActorMaterializer
+import akka.stream.scaladsl.{Sink, Source}
+
+class LorreBakerRightsOperations(resources: LorreAppConfig.ConfiguredResources) extends LazyLogging with IOLogging {
```

Comment:
why did you change name `BakingRights` -> `BakerRights`? also this class fetches endorsing rights

Reply:
I tried to find a naming that would generally include both baking and endorsing, so I tried to refrain from using exactly the "baking rights" word to make it more general (like operations related to what bakers do, where baker is a guy involved in baking blocks)

I'd like to find a good way to change this constant repetition of bakingEndorsingXXX